### PR TITLE
Add hardware support smoke testing instructions

### PR DESCRIPTION
**Pull request by seanbudd**
### Link to issue number:
Part of #16304

### Summary of the issue:

We do not have manual smoke testing notes for ensuring NVDA is supported on various forms of hardware.
This PR adds smoke testing notes for hardware

### CODE REVIEW

File: tests/manual/hardwareSupport.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Notes
+
+- Ensure all test results are documented, including screenshots or logs for failures.
+- Focus on identifying edge cases and compatibility issues during the migration process.
+- Prioritize testing on hardware configurations most commonly used by NVDA users.
+
+## Generic Steps
+
+1. Run the NVDA installer on the target hardware.
+2. Verify successful installation and basic functionality.
```
Comment (SaschaCowley) [inline tests/manual/hardwareSupport.md:17]: I think it might be helpful to be a little more specific here. A couple of specific tests could include checking typing into Notepad works as expected, using browsers (Chrome/Edge/Firefox) works, etc.

Comment (seanbudd) [reply tests/manual/hardwareSupport.md:17]: I intend to create a more thorough testing plan for applications.
I'm a bit worried about the scope of testing here - [as noted here](https://docs.google.com/document/d/1fPAAxst-rk5418eFS7_lzubXagaeyKoAjY8T1GBFHag/edit?tab=t.0) we have a long list of applications we want to test with.
Can you clarify what you think would be enough for confirming hardware support across devices? Just notepad and browsers? I think going through the full potential list for each device might be excessive.

Comment (SaschaCowley) [reply tests/manual/hardwareSupport.md:17]: Hmm, I get that. Perhaps just checking the NVDA GUI works as expected? So install and just poke around in NVDA's settings or something?

Comment (SaschaCowley) [reply tests/manual/hardwareSupport.md:17]: Does word come pre-installed these days? Possibly better to stick to notepad

Comment (seanbudd) [reply tests/manual/hardwareSupport.md:17]: I think in read-only mode you can use it without a license - if that works

Comment (SaschaCowley) [reply tests/manual/hardwareSupport.md:17]: I'm more interested in whether it's something else testers have to install. If we're trying to keep these tests quick and easy, we should try and use only in-box software

**Comment by codeofdusk** [comment]
A browser (Chrome) uses IA2, Win11 Notepad uses UIA. The NVDA GUI and run dialogue use MSAA. Testing all three quickly ensures functionality of major accessibility APIs.

File: tests/manual/hardwareSupport.md (hunk 1 above)
Comment (seanbudd) [reply tests/manual/hardwareSupport.md:17]:
```suggestion
2. Verify successful installation and basic functionality.
Test using Edge, Notepad and the NVDA GUI.
```

**Review by SaschaCowley — APPROVED** [review]
Thanks, @seanbudd
